apps/condo/domains/acquiring/schema/AcquiringIntegrationContext.js

```diff
@@ -66,6 +68,17 @@ const AcquiringIntegrationContext = new GQLListSchema('AcquiringIntegrationConte
         delete: false,
         auth: true,
     },
+    hooks: {
+        validateInput: ({ resolvedData, context, addValidationError }) => {
+            if (!hasDvAndSenderFields(resolvedData, context, addValidationError)) return
+            const { dv } = resolvedData
+            if (dv === 1) {
+                // NOTE: version 1 specific translations. Don't optimize this logic
```

Comment:
please use explicit returns

Reply:
Linter says it's redundant and we decided not to increase warning amount :)